### DISCUSSION TITLE
Tauri Monorepo

### DISCUSSION THREAD

**Discussion by FleetAdmiralJakob** [tauri-apps/tauri#6892] [Q&A]
Is there a way to use my shared Typescript backend with my Tauri app

**Comment by simonhyll** [comment]
Clarify what you mean by "Typescript backend"

If you mean your "node.js" backend then the answer is not directly as part of Tauri, no, Tauri doesn't deal with node.js, you can sidecar node.js into your app to run scripts but you'd have to set up communication and everything yourself

If you mean running your TS written code in the Rust backend then the answer is basically the same, Tauri doesn't give you a js runtime of any sort, so you'll have to use a crate and set it up manually yourself.
